telegram/ext/_utils/webhookhandler.py

```diff
@@ -31,11 +30,15 @@
 from tornado.httpserver import HTTPServer
 
 from telegram import Update
+from telegram._utils.logging import get_logger
 from telegram.ext._extbot import ExtBot
 
 if TYPE_CHECKING:
     from telegram import Bot
 
+# This module is not visible to users, so we log as Updater
```

Comment:
Just a quick understanding check: why as Updater?

Reply:
because Updater.run_webhook will use this class to start it